FirebaseDynamicLinks/Sources/FIRDynamicLink.m

```diff
@@ -134,6 +136,20 @@ + (FIRDLMatchType)matchTypeWithString:(NSString *)string {
   return [matchMap[string] integerValue] ?: FIRDLMatchTypeNone;
 }
 
++ (NSDictionary<NSString *, id> *)extractUtmParams:(NSDictionary<NSString *, id> *)parameters {
+  NSMutableDictionary<NSString *, id> *utmParamsDictionary = [[NSMutableDictionary alloc] init];
+
+  if (parameters) {
```

Comment:
I don't think the `if` statement is necessary here, the `for` loop won't run if `parameters` is `nil`.

Reply:
Removed